src/frontend/Components/Footer/Footer.css

```diff
@@ -0,0 +1,9 @@
+.footer{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: black;
+    width:100%;
+    height:60px;   /* Height of the footer */
```

Comment:
use rem unit so that you achieve responsiveness

Reply:
sure